### PR TITLE
added send to kindle

### PR DESCRIPTION
**Pull request by bearmannl**
Added the Amazon SendToKindle application, which can be used to send local files (such as PDF) to your Kindle device.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/13849)

### CODE REVIEW

File: manifests/a/Amazon/SendToKindle/1.1.1.250/Amazon.SendToKindle.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: Amazon.SendToKindle
+PackageVersion: 1.1.1.250
+MinimumOSVersion: 10.0.0.0
```
Comment (OfficialEsco) [inline manifests/a/Amazon/SendToKindle/1.1.1.250/Amazon.SendToKindle.installer.yaml:4]:
```suggestion
MinimumOSVersion: 10.0.0.0
InstallerSuccessCodes:
  - 1223
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VC++2015Redist-x86
      MinimumVersion: 14.0.23026.0
```

File: manifests/a/Amazon/SendToKindle/1.1.1.250/Amazon.SendToKindle.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+Installers:
+  - Architecture: x86
+    InstallerType: exe
```
Comment (OfficialEsco) [inline manifests/a/Amazon/SendToKindle/1.1.1.250/Amazon.SendToKindle.installer.yaml:11]:
```suggestion
    InstallerType: nullsoft
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
The error you were seeing is "Installer failed with exit code: 1223" which means Cancel by user which happens with these package installers..
The package also requires VC++2015 x86 which will be supported soon
The installer should also be nullsoft like before instead of exe :)

**Comment by OfficialEsco** [comment]
@bearmannl look at my comment/suggested changes

**Comment by bearmannl** [comment]
Thanks for the feedback @OfficialEsco

**Comment by denelon** [comment]
https://github.com/microsoft/winget-cli/issues/163 Support for Package Dependencies

**Comment by bearmannl** [comment]
re-worked installer and added version 1.1.0.252

**Comment by OfficialEsco** [comment]
Hey @bearmannl isn't this already submitted under https://github.com/microsoft/winget-pkgs/tree/master/manifests/a/Amazon/Kindle ?

**Pull request merged**